app/views/admin/participants/induction_records/edit_training_status.html.erb

```diff
@@ -0,0 +1,16 @@
+<% content_for :before_content, govuk_back_link(text: "Back", href: admin_participant_path(@participant_profile)) %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-l">Change the training status for this participant at <%= @induction_record.school_name %></h1>
+    <%= form_for @induction_record, url: update_training_status_admin_participant_induction_records_path(@participant_profile, @induction_record), method: :put do |f| %>
+      <%= f.govuk_collection_radio_buttons(
+              :training_status,
+              training_statuses,
+              :id,
+              :value,
+              legend: { text: "Select training status", tag: 'h1', size: 'l' },
+              options: { selected: @induction_record.training_status }) %>
+      <%= f.govuk_submit "Submit" %>
+    <% end %>
```

Comment:
We’ve currently got 2 `<h1>` tags - we can combine these into a single tag which is also the legend. 

I also think it’s worth including the participant name as well as the school - just to make double-sure that people are updating the right person!

```suggestion
    <%= form_for @induction_record, url: update_training_status_admin_participant_induction_records_path(@participant_profile, @induction_record), method: :put do |f| %>
      <%= f.govuk_collection_radio_buttons(
              :training_status,
              training_statuses,
              :id,
              :value,
              legend: { text: "Training status for #{@participant_profile.name} at #{@induction_record.school_name}", tag: 'h1', size: 'l' },
              options: { selected: @induction_record.training_status }) %>
      <%= f.govuk_submit "Save" %>
    <% end %>
```